src/metaschema/oscal_control-common_metaschema.xml

```diff
@@ -205,6 +213,14 @@
         </define-assembly>
         <define-assembly name="parameter-guideline">
                 <formal-name>Guideline</formal-name>
+                <!-- 
+                        TODO: Is `@ns` flag enough or should we add system for
+                        dictating guidelines?
+
+                        Use cases: RMF recommends one thing, maybe different with
+                        FedRAMP, then a contradiction or subset of constraints is
+                        required, not recommended for example FCEB agency (Important Federal Agency).
+                -->
```

Comment:
@aj-stein-nist  Please create a new issue for this use case.

Reply:
See #54.

Reply:
Done, follow-up issue is https://github.com/usnistgov/OSCAL/issues/1410.